coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "covered_percent": 88.24
```

Comment:
How did coverage go down 28 percent?

Reply:
I figured it out - I didn't have chrome installed so a bunch of view testing was getting ignored.

Reply:
Oops! The readme _should_ have all the dependencies listed but it's very sparse.